lib/routes/assets.js

```diff
@@ -35,22 +40,25 @@ module.exports = function (app) {
     app.bffs.build(fingerprint, gzip, function (err, build) {
       if (err) {
         return res.status(500).send(comment('Failed to lookup the build: ' + err.message, {
-          extension: extension
+          extension
         }));
       }
 
       if (!build) {
         return res.status(404).send(comment('Build not found', {
-          extension: extension
+          extension
         }));
       }
 
-      if (gzip) {
-        res.setHeader('Content-Encoding', 'gzip');
-      }
-
-      res.setHeader('Content-Length', build.source.length);
-      res.send(build.source);
+      var url = extension === '.map' ? build.url + '.map' : build.url;
```

Comment:
not in how we publish currently, it always lives in the same `fingerprint` bucket as the source for them to work properly in browsers.